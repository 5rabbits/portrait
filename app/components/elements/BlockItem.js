import React, { PureComponent } from 'react';

export default class BlockItem extends PureComponent {
  render() {
    return (
      <div className="block-item">

        <label className="switch mini pull-right">
          <input type="checkbox"/>
          <span></span>
        </label>
        <strong>App movil</strong>
        <span className="type-caption color-secondary ml-xs">Sosafe</span>

      </div>
    );
  }
}
