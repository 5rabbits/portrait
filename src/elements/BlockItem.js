import React from 'react'

const BlockItem = () =>
  <div className="block-item">
    <label className="switch mini pull-right">
      <input type="checkbox" />
      <span />
    </label>
    <strong>App movil</strong>
    <span className="type-caption color-secondary ml-xs">Sosafe</span>
  </div>

export default BlockItem
