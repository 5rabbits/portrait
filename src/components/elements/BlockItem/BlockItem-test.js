import { React, toJSON, mount } from 'test/helper'
import BlockItem from 'elements/BlockItem'

describe('BlockItem', () => {
  it('should render with minimum props', () => {
    const component = mount(<BlockItem />)

    expect(toJSON(component)).toMatchSnapshot()
  })
})
