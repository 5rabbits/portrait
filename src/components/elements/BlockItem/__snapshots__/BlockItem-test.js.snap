// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockItem should render with minimum props 1`] = `
<BlockItem>
  <div
    className="block-item"
  >
    <label
      className="switch mini pull-right"
    >
      <input
        type="checkbox"
      />
      <span />
    </label>
    <strong>
      App movil
    </strong>
    <span
      className="type-caption color-secondary ml-xs"
    >
      Sosafe
    </span>
  </div>
</BlockItem>
`;
